client/src/App.js

```diff
@@ -1,11 +1,96 @@
-import React, {useEffect, useState} from 'react';
-import logo from './logo.svg';
-import Ticket from './components/Ticket'
+import React, { useEffect, useState } from 'react';
+import axios from 'axios';
+import Ticket from './components/Ticket';
 import './App.css';
+import Search from './components/Search';
+import Header from './components/Header';
 
 function App() {
+  const [counter, setCounter] = useState(0); // counter of hidden tickets
+  const [tickets, setTickets] = useState([]); // list of tickets
+  const [call, setCall] = useState(0); // state that will be passed as a prop to restore tickets
+  const [scrollDisplay, setScrollDisplay] = useState('none'); // state that will be used to show scroll up button
```

Comment:
Better use a boolean then a CSS property.
`showScrollButton = true || false;`
`{display: showScrollButton ? "block" : "none"}`
or better yet, use `hidden` attribute

Reply:
Fixed, but why is that better? the result is the same and the number of renders is the same.